README.md

```diff
@@ -35,6 +35,15 @@ This project is in alpha status.
 
 See currently open [issues](https://github.com/Bee-Lab/bowerphp/issues).
 
+Contributing
+------------
+
+All contribution are welcome, just take a look at our [issues](https://github.com/Bee-Lab/bowerphp/issues) tracker if you wanna start somewher
```

Comment:
Well well, don't know how those 2 slipped in, but I am sure it can't be related to sleep deprivation...